i18n/en/code.json

```diff
@@ -0,0 +1,423 @@
+{
+  "untranslated-page-text": {
```

Comment:
Question:  curious, how do we get these keys?

Reply:
These were automatically generated by running
```
npm run write-translations
```

Reply:
I see, thanks!